CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ a release.
 
 ### Fixed
 - Tree: Creation of dynamic `Node::$sibling` property, which is deprecated as of PHP >= 8.2
+- Sluggable: Use `TranslationWalker` hint when looking for similar slugs (`getSimilarSlugs` method) for entities which implement `Translatable` interface (#100, #2530)
```

Comment:
The changelog entry can be moved up to the unreleased section near the top, this definitely isn't part of the already tagged 3.12 release 😅